edx_reverification_block/xblock/tests/test_reverification_block.py

```diff
@@ -53,3 +58,28 @@ def test_load_student_view(self, xblock):
         DummyReverificationService.get_status.return_value = "submitted"
         xblock_fragment = self.runtime.render(xblock, "student_view")
         self.assertTrue('submitted' in xblock_fragment.body_html())
+
+        # User skip the re-verification
+        data = json.dumps({'checkpoint': 'FinalExam', 'user_id': 5, "course_id": "edX/Demo/Course"})
+        resp = self.request(xblock, 'skip_verification', data, response_format='json')
+        self.assertTrue(resp.get('result'))
+
+        attrs['is_user_skipped_reverification.return_value'] = True
+        DummyReverificationService.configure_mock(**attrs)
+        self.runtime._services['reverification'] = DummyReverificationService
+        xblock_fragment = self.runtime.render(xblock, "student_view")
+
+        self.assertTrue("You have skipped re-verification" in xblock_fragment.body_html())
+
+    @scenario(TESTS_BASE_DIR + '/data/basic_scenario.xml')
+    def test_load_studio_view(self, xblock):
+        """
+        Reverification XBlock basic test for verifying that the user gets some
+        HTML.
+        """
+        # Test that creating Reverification XBlock for the first time user gets
```

Comment:
Make this docstring: `Test that the user gets warning message for configuring the Reverification XBlock for the first time`